jwql/utils/constants.py

```diff
@@ -92,3 +92,11 @@
                               'NRCB1', 'NRCB2', 'NRCB3', 'NRCB4']
 
 NIRCAM_LONGWAVE_DETECTORS = ['NRCA5', 'NRCB5']
+
+AMPLIFIER_BOUNDARIES = {'nircam': {'1': [(0, 0), (512, 2048)], '2': [(512, 0), (1024, 2048)],
+                                   '3': [(1024, 0), (1536, 2048)], '4': [(1536, 0), (2048, 2048)]}
+                        }
```

Comment:
Oops. Yeah, I forgot to do that. I was a little nervous because NIRISS' detector is rotated 90 degrees compared to NIRCam, and I wasn't sure how the code would react to 4 horizontal blocks on the detector rather than 4 vertical blocks.

Reply:
I've asked Kevin Volk to confirm the details of NIRISS' amplifier readouts.

Reply:
I've also asked the same of the MIRI/NIRSpec teams